Resources/config/services.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <service
+            id="handcrafted_in_the_alps_sulu_resource.common.doctrine_list_representation_factory"
+            class="HandcraftedInTheAlps\Bundle\SuluResourceBundle\ListRepresentation\DoctrineListRepresentationFactory"
+        >
+            <argument type="service" id="sulu_core.doctrine_rest_helper"/>
+            <argument type="service" id="sulu_core.list_rest_helper"/>
+            <argument type="service" id="sulu_core.doctrine_list_builder_factory"/>
+            <argument type="service" id="sulu_core.list_builder.field_descriptor_factory"/>
+        </service>
+
+        <service
+            id="v.common.doctrine_nested_list_representation_factory"
+            class="HandcraftedInTheAlps\Bundle\SuluResourceBundle\ListRepresentation\DoctrineNestedListRepresentationFactory"
+        >
+            <argument type="service" id="sulu_core.doctrine_rest_helper"/>
+            <argument type="service" id="sulu_core.doctrine_list_builder_factory"/>
+            <argument type="service" id="sulu_core.list_builder.field_descriptor_factory"/>
+            <argument type="service" id="doctrine.orm.entity_manager"/>
+        </service>
+
+        <service
+            id="handcrafted_in_the_alps_sulu_resource.handler_failed_exception_listener"
+            class="HandcraftedInTheAlps\Bundle\SuluResourceBundle\EventSubscriber\HandlerFailedExceptionSubscriber"
```

Comment:
I think this should be a middleware and not a handlerfailedexception subscriber else you can still not catch exception inside the controller